content/zh-cn/docs/contribute/style/content-organization.md

```diff
@@ -66,7 +65,7 @@ The `Documentation` main menu is built from the sections below `docs/` with the
 -->
 ### 文档主菜单
 
-`文档` 主菜单是从 `docs/` 下面的章节构建的。
+`文档`主菜单是从 `docs/` 下面的章节构建的。
```

Comment:
Maybe it would be better. (It's up to the PR author)
```suggestion
“文档”主菜单是从 `docs/` 下面的章节构建的。
```

Reply:
Not sure. Or we can just follow the en source:

The `Documentation` main menu is built from the sections below `docs/` ...